charts/pxc-db/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: "1.6.0"
 description: A Helm chart for installing Percona XtraDB Cluster Databases using the PXC Operator.
 name: pxc-db
 home: https://www.percona.com/doc/kubernetes-operator-for-pxc/kubernetes.html
-version: 0.1.11
+version: 0.1.12
```

Comment:
Please increment this to 0.1.13 since there was already one other PR merged which incremented to 0.1.12.

Reply:
done